src/locales/en/translation.json

```diff
@@ -0,0 +1,127 @@
+  "options_token_dialog_description": "Your token will only be used to get avatar and username and hypertrons.json from any Github repositories, which needs no more scopes than basic `public_repo`. The token will be keep in local storage and will not be sent to any other domains except github.com.",
+  "options_token_dialog_checking": "Checking",
+  "options_token_dialog_error": "Bad token",
+  "options_about_title": "About",
+  "options_about_toolTip": "About HyperCRX",
+  "options_about_description": "HyperCRX is a browser extension to get more insights into projects and developers on GitHub.",
+  "hypertrons_tab_title": "Hypertrons Commands",
+  "hypertrons_tab_description": "You can use hypertrons commands here. See usage:",
+  "hypertrons_tab_link": "https://www.hypertrons.io/#/component/index",
+  "header_label_activity": "Activity",
+  "header_label_OpenRank": "OpenRank",
+  "header_label_participant": "Participants",
+  "header_label_contributor": "Contributors",
+  "fork_popup_title": "Fork Events",
+  "star_popup_title": "Star Events",
+  "issue_popup_title": "Issue Open, Close and Comment Events",
+  "pr_popup_title": "Open PR, PR Merge, Review Comment Events",
+  "merged_lines_popup_title": "Merged Addition & Deletion Code Lines",
+  "docs_gpt_chat_widget": "OSS-GPT Chat Bot",
+  "OSS_GPT_subtitle": "Ask anything about %v",
+  "OSS_GPT_subtitle_notAvailable": "NOT AVAILABLE for %v",
+  "OSS_GPT_welcome": "Hi, I'm an assistant powered by [DocsGPT](https://github.com/arc53/docsgpt) and [X-lab](https://github.com/X-lab2017). Ask me anything about %v!",
+  "OSS_GPT_notAvailable": "OSS-GPT currently is **NOT AVAILABLE** for %v. If you want docs support for the repository, please check [this issue](https://github.com/hypertrons/hypertrons-crx/issues/609) and make a request there :)\n\nSee [all available docs](https://oss.x-lab.info/hypercrx/docsgpt_active_docs.json)\n\nThis chat widget can also be enabled/disabled in the extension options page whenever you want.",
+  "issue_icon": "GitHub repository issue count over time. And you can zoom in/out on the axes using the mouse scroll wheel.",
+  "pr_icon": "GitHub repository pr count over time. And you can zoom in/out on the axes using the mouse scroll wheel.",
+  "fork_icon": "GitHub repository fork count over time. And you can zoom in/out on the axes using the mouse scroll wheel.",
+  "star_icon": "GitHub repository star count over time. And you can zoom in/out on the axes using the mouse scroll wheel.",
+  "activity_icon": "GitHub repository activity count over time. And you can zoom in/out on the axes using the mouse scroll wheel.",
+  "openrank_icon": "GitHub repository openrank count over time. And you can zoom in/out on the axes using the mouse scroll wheel.",
+  "contributors_participants_icon": "GitHub repository contributors and participants count over time. And you can zoom in/out on the axes using the mouse scroll wheel.",
```

Comment:
These are duplicate. Could you look into [i18n nesting](https://www.i18next.com/translation-function/nesting) to simplify this part?